resources/user-agents/browsers/mobile-safari-uiwebview/mobile-safari-uiwebview.json

```diff
@@ -66,15 +73,30 @@
             "iPod": "iPod Touch"
```

Comment:
There's a test that fails if I do that split:

>Mozilla/5.0 (iPhone; CPU OS 9_3_5 like Mac OS X; en-US) 

https://github.com/browscap/browscap/blob/master/tests/fixtures/issues/issue-1323.php#L481

I may be able to at least leave the `iOS_A` patterns off of iPad, but leave the `iOS_C` patterns on iPhone/iPod.

Of all of the `iOS_C` patterns we saw on iPhone devices this year, that style of agent is a majority of them (that short agent that is tested there in 1323).

For example, small chunk:

>Mozilla/5.0 (iPhone; CPU OS 9_2 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 9_1 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 8_3 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 9_3 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 8_4 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 9_0 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 7_1 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 9_2 like Mac OS X; en-JP)
Mozilla/5.0 (iPhone; CPU OS 9_3 like Mac OS X; en-US)
Mozilla/5.0 (iPhone; CPU OS 7_0 like Mac OS X; ja-JP)
Mozilla/5.0 (iPhone; CPU OS 9_3 like Mac OS X; ja-KR)
Mozilla/5.0 (iPhone; CPU OS 9_2 like Mac OS X; ja-DZ)
Mozilla/5.0 (iPhone; CPU OS 9_2 like Mac OS X; ja-AU)
Mozilla/5.0 (iPhone; CPU OS 9_2 like Mac OS X; en-US)
Mozilla/5.0 (iPhone; CPU OS 9_1 like Mac OS X; ja-US)
Mozilla/5.0 (iPhone; CPU OS 9_1 like Mac OS X; en-US)
Mozilla/5.0 (iPhone; CPU OS 9_1 like Mac OS X; en-JP)
Mozilla/5.0 (iPhone; CPU OS 9_1 like Mac OS X; en-GB)
Mozilla/5.0 (iPhone; CPU OS 8_4 like Mac OS X; en-US)
Mozilla/5.0 (iPhone; CPU OS 8_3 like Mac OS X; zh-CN)
Mozilla/5.0 (iPhone; CPU OS 8_3 like Mac OS X; es-CR)
Mozilla/5.0 (iPhone; CPU OS 8_3 like Mac OS X; en-US)
Mozilla/5.0 (iPhone; CPU OS 8_3 like Mac OS X; en-JP)

The UI Webview file doesn't have versions at all, so it's probably not a huge deal to leave these since they're only replicated 3 times, and may be a good fallback.